src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.tsx

```diff
@@ -29,6 +33,17 @@ function BaseSidebarScreen() {
         Timing.start(CONST.TIMING.SIDEBAR_LOADED, true);
     }, []);
 
+    const [activeWorkspace] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${activeWorkspaceID ?? -1}`);
```

Comment:
NAB. Move this to be before the `useEffect`

Reply:
I moved